README.md

```diff
@@ -501,6 +521,9 @@ Available controllers:
 - `fts_broadcaster[force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster]`
 - `joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster]`
 
+Notes:
+  - Wrench messages are not displayed properly in Rviz as NaN values are not handled in Rviz and FTS Broadcaster may send NaN values.
```

Comment:
I believe we added a tickbox to the rviz plugin, once ticked, rviz will not freak out on NaN values

Reply:
Thanks for pointing that out. I have cleaned up the Readme so that only the comments related to this gazebo rrbot update are in it.